### DISCUSSION TITLE
Fluent Mapping Property

### DISCUSSION THREAD

**Discussion by alberk8** [linq2db/linq2db#4761] [Q&A]
Is there a way to automatically skip property matching if it does exist in the POCO or in the actual SQL database.
In the example class below let say that Age does not exist in the SQL Database and PostCode exist in the Database.   This will fail to run when I try to query the table named Person with a select all.
 
Example POCO

```
public class Person 
{
    public string Name {get;set;}
    public string Address {get;set;}
    public int Age {get;set;}
} 
```

**Comment by MaceWindu** [comment]
You need to mark `Age` column with `NotColumn` attribute.
In fluent mapping it will be
```cs
.Property(e => e.Age).IsNotColumn()
```
